libs/components/src/next/form/checkbox/vl-checkbox.component.cy.ts

```diff
@@ -0,0 +1,300 @@
+import { html } from 'lit';
+import { registerWebComponents } from '@domg-wc/common-utilities';
+import { VlCheckboxComponent, CheckboxDefaults } from './index';
+
+registerWebComponents([VlCheckboxComponent]);
+export type CheckboxDefaultTypes = Partial<typeof CheckboxDefaults>;
+
+const mountCheckbox = ({ name, value, isSwitch }: CheckboxDefaultTypes) => {
+    cy.mount(html` <vl-checkbox-next name=${name} value=${value} ?switch=${isSwitch}>Bevestig.</vl-checkbox-next> `);
+};
+const mountCheckboxInForm = ({ value, isSwitch, checked }: CheckboxDefaultTypes) => {
```

Comment:
Ik zou deze test op termijn in de e2e tests van de playgrounds plaatsen en hier de testen puur op component niveau houden.

Reply:
ok